tools/Github/CreatePR.ps1

```diff
@@ -64,4 +64,11 @@ $Description
 * **SHOULD NOT** adjust version of module manually in pull request
 "@
 $RequestBody = @{"title" = $Title; "body" = $PrBody; "head" = $HeadBranch; "base" = $BaseBranch }
-Invoke-WebRequest -Uri https://api.github.com/repos/Azure/azure-powershell/pulls -method POST -Headers $Headers -Body ($RequestBody | ConvertTo-Json)
\ No newline at end of file
+$Uri = "https://api.github.com/repos/Azure/azure-powershell/pulls"
+$PullRequests = Invoke-WebRequest -Uri $Uri -Headers $Headers | ConvertFrom-Json
+$ExistingPr = $PullRequests | Where-Object { $_.title -eq $Title }
```

Comment:
I mean add a switch parameter such as `FailOnPRExisted` and set the default value `False`. If other script care about the existing of PR, they can get the error. And script will be success for those who doesn't care about.